pkg/api/v1/helper/helpers.go

```diff
@@ -23,12 +23,18 @@ import (
 
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
-
 	"k8s.io/api/core/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/helper"
 )
 
+const (
+	MODIFIED  = "modified"
```

Comment:
These are needed for the reflecting node changes. So, I guess these are needed here.